### PR TITLE
Release Proposal: v2.14

### PR DESCRIPTION
**Pull request by mattmcneeney**
Announcing a new release candidate as described in the [Release Process](https://github.com/openservicebrokerapi/servicebroker/blob/master/CONTRIBUTING.md#release-process).

**Release version**: v2.14
**Release SHA**: a233c97d260c00c656feb7fa666ba9b3d6433b64 
**Target release date**: 2018-07-24

The one-week [Review Process](https://github.com/openservicebrokerapi/servicebroker/blob/master/CONTRIBUTING.md#review-process) will be triggered on the next weekly call.

### CODE REVIEW

**Comment by fmui** [comment]
The ToC contains a link to the 2.12 changes section (`[Changes Since v2.12](#changes-since-v212)`), which should be a 2.13 changes section now.

**Comment by mattmcneeney** [comment]
Thanks @fmui ! Fixed.

Reminder: The one week review deadline for this is tomorrow!
cc @openservicebrokerapi/osbapi-pmc

File: openapi.yaml [hunk 1]
```diff
@@ -85,7 +85,7 @@ paths:
           content:
             application/json:
               schema:
-                $ref: '#/components/schemas/ServiceInstanceAsyncProvision'
+                $ref: '#/components/schemas/ServiceInstanceAsyncOperation'
```
Comment (duglin) [inline openapi.yaml]: Why are these changes part of this PR and not part of some other PR? Were they just missed or is there something in the release process that causes these changes to only happen now?

**Comment by duglin** [comment]
LGTM

**Comment by fmui** [comment]
LGTM

**Comment by pmorie** [comment]
Lgtm

Looks like travis is broken?

**Comment by mattmcneeney** [comment]
@pmorie it's always broken when we do releases, since the changes contain URLs that don't exist yet!

**Comment by mattmcneeney** [comment]
> A release proposal requires at least 4 "LGTM" comments from at least 4 different organizations to be approved.

This release candidate has been open now for more than one week and has the required LGTMs. I will now merge this and follow the [rest of the release process](https://github.com/openservicebrokerapi/servicebroker/blob/master/CONTRIBUTING.md#once-approved)!

**Pull request merged**